fixedwindow_test.go

```diff
@@ -102,3 +103,21 @@ func (s *LimitersTestSuite) TestFixedWindowOverflow() {
 		})
 	}
 }
+
+func BenchmarkFixedWindows(b *testing.B) {
+	s := new(LimitersTestSuite)
+	s.SetT(&testing.T{})
+	s.SetupSuite()
+	capacity := int64(1)
+	rate := time.Second
+	clock := newFakeClock()
+	windows := s.fixedWindows(capacity, rate, clock)
+	for name, window := range windows {
+		b.Run(name, func(b *testing.B) {
+			for i := 0; i < b.N; i++ {
+				_, _ = window.Limit(context.TODO())
```

Comment:
Consider adding the error check (here and below)

Reply:
Added error check